### PR TITLE
Add `The Followpoint: DigitalHypno, the osu! Professor` news post

### PR DESCRIPTION
**Pull request by minusQuantumNeko**
[web preview](https://osu2.roanh.dev/home/news/2024-04-04-the-followpoint-digitalhypno-the-osu-professor)

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+>
+> I've been a bit less active lately as I'm focused on making a new YouTube series, which I'll talk more about later in this interview.
+
+*How did you get introduced to osu! and what was your first impression of it?*
+
+> I first came across osu! through League of Legends livestreams. At the time, many streamers would play a few maps while waiting in queue. At first, I thought it was just another random queue game and didn't think much of it.
+>
+> One night in particular—May 11, 2014—I was watching a League streamer play osu!, and the chat was going absolutely crazy over the streamer's performance. I thought to myself, "This game doesn't look *that* hard," and I made my osu! account that same day.
+
+*What is the story behind the symmetrical play count graph in your profile?*
```
Comment (Walavouchey) [inline news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md:35]: should include an image

File: news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+>
+> To make sure I ended the month with the right amount of play count, I just played normally until the last few days of the month, at which point I would gain a lot of play count quickly by retrying maps over and over. osu! has a minimum score requirement for a play to count towards your play count, so it wasn't as simple as just pressing the retry button over and over. I mostly did this "play count farming" in Catch the Beat, since lower star maps in that gamemode don't require as much physical strain or focus as other game modes, which is important when you're trying to gain over 1,000 play count in one sitting.
+>
+> There was one time where I even flagged osu!'s auto-restriction for gaining play count too quickly, causing my account to be restricted for a few hours and disabled for a few minutes. Shoutout to [Ephemeral](https://osu.ppy.sh/users/102335) for resolving my case quickly and for all the other work he's done for the community.
```
Comment (Walavouchey) [inline news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md:41]: any mention of restrictions has historically been a no-go

File: news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+>
+> Rest assured, my name on osu! will always be DigitalHypno. I don't mind being called honmi, Hypno, or Habib, which is my real name.
+
+*Do you believe the hardware we have access to today has reached its peak, or do you think people are able to develop better tools for easier and faster improvement?*
+
+> As always, it's hard to imagine how new hardware will impact the community. Who could've predicted rapid trigger's widespread adoption in the osu! community even 5 years ago?
+>
+> That said, I do think we've reached a point where the bottleneck to a player's performance truly is their actual skill rather than their hardware. I can't imagine any changes in the hardware meta that would change this fact, but we can check back in another 5 years.
+
+*You've got a lot of badges from tournaments, what made you start playing tournaments and do you feel like you have encouraged others to start playing tournaments as well?*
```
Comment (Walavouchey) [inline news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md:57]:
```suggestion
*You've got a lot of badges from tournaments. What made you start playing tournaments and do you feel like you have encouraged others to start playing tournaments as well?*
```

File: news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+>
+> As I became a competitive player in open-rank tournaments, I definitely became one of the most infamous derankers for many years. Along with the rest of the community, though, I definitely recognized the issue. I think the whole point of rank restricted tournaments is to allow for a more level playing field for players around those ranks, but the problem was that while tournaments were seeded based on solo rank, solo rank isn't based on tournament performance at all.
+>
+> To help solve this problem, I designed a seeding system in 2018 called Badge Weighted Seeding (BWS), a seeding formula that artificially increases your rank based on how many tournament badges you have. Tournament badges come from winning tournaments, so the idea is that the more badges you have, the more likely it is that you shouldn't be playing in rank restricted tournaments anymore. Despite not being a perfect solution, I believe it solves a lot of the problems that used to exist.
```
Comment (Walavouchey) [inline news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md:73]:
```suggestion
> To help solve this problem, I designed a seeding system in 2018 called [Badge Weighted Seeding (BWS)](/wiki/Tournaments/Badge-weighted_seeding), a seeding formula that artificially increases your rank based on how many tournament badges you have. Tournament badges come from winning tournaments, so the idea is that the more badges you have, the more likely it is that you shouldn't be playing in rank restricted tournaments anymore. Despite not being a perfect solution, I believe it solves a lot of the problems that used to exist.
```

File: news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+>
+> There are some other, more polarizing changes in the tournament scene, such as "slot pooling" (mappools where each map covers a very predictable skillset, such as the first No Mod map being aim and the second No Mod map being streams), but overall, I think the tournament scene has really found its footing in the last few years.
+
+*In the future, do you think it's possible for osu! tournaments to become mainstream e-sport events?*
+
+> It's absolutely possible, especially with recent developments such as the [Tournament Committee](/wiki/People/Tournament_Committee), a group of seasoned tournament staff members that work to improve the quality of osu! tournaments, and people like BTMC that are always finding ways to take competitive osu! to the next level.
```
Comment (Walavouchey) [inline news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md:87]:
```suggestion
> It's absolutely possible, especially with recent developments such as the [Tournament Committee](/wiki/People/Tournament_Committee), a group of seasoned tournament staff members that work to improve the quality of osu! tournaments, and people like [BTMC](https://osu.ppy.sh/users/3171691) that are always finding ways to take competitive osu! to the next level.
```

File: news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+> I'm currently inactive on osu! as I'm working on a new YouTube game show series called Social Circle. Inspired by [The Genius](https://en.wikipedia.org/wiki/The_Genius_(TV_series)), Social Circle is a last-man-standing strategy game show where 13 players play a series of strategic, social, and psychological games, eliminating one player per episode until only one remains. It's played in Minecraft using proximity voice to simulate a real-world environment, but the strategy games themselves do not involve Minecraft mechanics in any way.
+>
+> The players are all from the osu! community, with a wide mix of personalities and some familiar faces. It'll be at least a few months until it comes out, but I'm sure it'll be worth the wait. Please check it out when it's released!
+>
+>I'm sure I'll be back to playing osu! more regularly one day. This community will always feel like home to me, and as with most people who have been around here long enough, I'm forever indebted to this game for all the people I've met and all the experiences I've had. Thank you for the interview!
```
Comment (Walavouchey) [inline news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md:97]:
```suggestion
> I'm sure I'll be back to playing osu! more regularly one day. This community will always feel like home to me, and as with most people who have been around here long enough, I'm forever indebted to this game for all the people I've met and all the experiences I've had. Thank you for the interview!
```

File: news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+>
+> My final words: Go play [osu!lazer](https://osu.ppy.sh/home/download)! It's much better than you might think.
+
+*Bonus question: Can you cook a steak?*
```
Comment (Walavouchey) [inline news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md:101]: colons aren't like periods, as in they don't end sentences

```suggestion
*Bonus question: can you cook a steak?*
```

File: news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+> The winner of mangomizer vs BTMC can face off against me.
+
+---
+
```
Comment (Walavouchey) [inline news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md:106]:
```suggestion

For the three people who are out of the loop regarding who this "BTMC" person mentioned here is, you'll probably want to read [his interview](https://osu.ppy.sh/home/news/2024-03-13-the-followpoint-btmc-the-face-of-osu). More likely though, if you're out of the loop regarding this "steak" business, [this YouTube video](https://www.youtube.com/watch?v=0B6TyWML0zg) (and its comment section) may clarify that.

```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+layout: post
+title: "The Followpoint: DigitalHypno, the osu! Professor"
+date: 2024-04-04 15:00:00 +0000
+---
+
+Make sure to join us for an interview with DigitalHypno, the famous osu! tutor and content creator.
+
+![](/wiki/shared/news/2024-04-04-the-followpoint-digitalhypno-the-osu-professor/banner.jpg)
+
+Welcome or welcome back to The Followpoint, a "Community Spotlight" series where we interview various players and community figures to get to know them better!
```
Comment (Walavouchey) [inline news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md:11]:
```suggestion
Welcome or welcome back to The Followpoint, a "community spotlight" series where we interview various players and community figures to get to know them better!
```

File: news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+date: 2024-04-04 15:00:00 +0000
+---
+
+Make sure to join us for an interview with DigitalHypno, the famous osu! tutor and content creator.
+
+![](/wiki/shared/news/2024-04-04-the-followpoint-digitalhypno-the-osu-professor/banner.jpg)
+
+Welcome or welcome back to The Followpoint, a "community spotlight" series where we interview various players and community figures to get to know them better!
+
+Today we are joined by DigitalHypno (a.k.a. honmi), who is known for their achievements and contributions, such as winning many tournaments, their content about osu!, as well as their contributions to Project Loved, Beatmap Spotlights and the osu! tournaments scene.
```
Comment (Walavouchey) [inline news/2024/2024-04-04-the-followpoint-digitalhypno-the-osu-professor.md:13]:
```suggestion
Today we are joined by [DigitalHypno](https://osu.ppy.sh/users/4384207) (a.k.a. honmi), who is known for their achievements and contributions, such as winning many tournaments, their content about osu!, as well as their contributions to Project Loved, Beatmap Spotlights and the osu! tournaments scene.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
i'm usually not too picky to proofread interviewee answers but digitalhypno said it's ok

**Review by DigitalHypno — COMMENTED** [review]
some minor wording changes

**Review by peppy — APPROVED** [review]

**Pull request merged**